OpenQA/Isotovideo/Utils.pm

```diff
@@ -197,7 +197,7 @@ sub handle_generated_assets ($command_handler, $clean_shutdown) {
     # mark hard disks for upload if test finished
     return unless $bmwqemu::vars{BACKEND} =~ m/^(qemu|generalhw)$/;
     my @toextract;
-    my $nd = $bmwqemu::vars{NUMDISKS};
+    my $nd = defined $bmwqemu::vars{NUMDISKS} ? $bmwqemu::vars{NUMDISKS} : 1;
```

Comment:
Could you add a unit test for this?

Reply:
This requirement to test any fix is kind of punishment for people who fix problems :).

Reply:
I made a one-line change to rely on the default in `t/14-isotovideo.t`. That's enough to cover it.